src/test-run/commands/index.js

```diff
@@ -188,6 +188,23 @@ export class TypeTextCommand extends Assignable {
     }
 }
 
+export class NavigateToCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type = TYPE.navigateTo;
+        this.url  = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'url', type: nonEmptyStringArgument, required: true }
```

Comment:
> We just call the hammerhead.navigateTofunction , I suppose it should be tested in hammerhead

I have no idea. Also, I have no idea what will happen if we pass incorrect URL that's why I want tests which will describe this behavior (and help us avoid regressions).